docs/source/api/plugin/usage-reporting.md

```diff
@@ -150,7 +129,12 @@ For example, you can look at the GraphQL operation's name in `requestContext.ope
 </td>
 <td>
 
-By default, this plugin associates client information such as name and version with user requests based on HTTP headers starting with `apollographql-client-`. If you have another way of communicating client information to your server, tell the plugin how it works with this option. This option has type `(GraphQLRequestContext) => { clientName?: string, clientVersion?: string, clientReferenceId?: string }`.
+Specify this function to provide Apollo Studio with client details for each processed request. Apollo Studio uses this information to [segment metrics by client](https://www.apollographql.com/docs/studio/client-awareness/).
+
+This function is passed a [`GraphQLRequestContext`](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-server-types/src/index.ts#L115-L150) object containing all available information about the request. It should return an object with `clientName` and `clientVersion` fields that identify the associated client.
```

Comment:
Are you explicitly removing the reference to clientReferenceId? I think it isn't mentioned much in the docs. Maybe we should just get rid of the feature entirely? (I've always been skeptical of its value.)

Reply:
I am explicitly removing it with a similarly skeptical impression regarding its value. I'll double-check on the consequences of removing it from this doc enitrely

Reply:
I have double-checked and indeed it is good and sound to un-document it!